libs/ngu/carousel/src/lib/ngu-carousel/ngu-carousel.component.ts

```diff
@@ -339,89 +336,79 @@ export class NguCarousel<T>
   }
 
   ngOnDestroy() {
-    this._hammer?.destroy();
     this._destroy$.next();
   }
 
   /** Get Touch input */
   private _setupHammer(): void {
-    // Note: doesn't need to unsubscribe because streams are piped with `takeUntil` already.
-    this._nguCarouselHammerManager
-      .createHammer(this._touchContainer.nativeElement)
-      .subscribe(hammer => {
-        this._hammer = hammer;
-
-        hammer.get('pan').set({ direction: Hammer.DIRECTION_HORIZONTAL });
-
-        this._nguCarouselHammerManager.on(hammer, 'panstart').subscribe(() => {
-          this.carouselWidth = this._nguItemsContainer.nativeElement.offsetWidth;
-          this.touchTransform = this.transform[this.deviceType!]!;
-          this.dexVal = 0;
-          this._setStyle(this._nguItemsContainer.nativeElement, 'transition', '');
-        });
+    const options = {
+      "supportedGestures": [Pan]
+    };
 
-        if (this.vertical.enabled) {
-          this._nguCarouselHammerManager.on(hammer, 'panup').subscribe((ev: any) => {
-            this._touchHandling('panleft', ev);
-          });
+    this._pointerListener = new PointerListener(this._touchContainer.nativeElement, options);
```

Comment:
Just to understand if this property is used to hold the reference, should er also unsubscribe to this listener onDestroy

Reply:
No required, Removed